src/baseEditorComponent.tsx

```diff
@@ -1,7 +1,8 @@
 import React from 'react';
 import Handsontable from 'handsontable';
+import { HotEditorProps } from './types';
 
-class BaseEditorComponent<P = {}, S = {}, SS = any> extends React.Component<P, S, SS> implements Handsontable._editors.Base {
+class BaseEditorComponent<P = {}, S = {}, SS = any> extends React.Component<P | HotEditorProps, S> implements Handsontable._editors.Base {
```

Comment:
Very elegant solution 👏 Any idea why the `hot-editor` doesn't have to be on this props list? 🤔

Reply:
It think it has something to do with it being hyphenated. But it's a good idea to add it as a required prop - it might prevent some potential problems. 👍